Routes/login.route.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const route = express.Router();
+
+
+route.post('/', (req, res, next) => {
```

Comment:
why not include with the users routes

Reply:
It's a different url. You want to send a request to db/login,  and not db/users/login.